doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -5550,125 +5679,24 @@ equivalences are indeed taken into account, otherwise only single
        statement, including hypotheses, features a ``=1`` and whose name
        contains the string ``bij``.
 
-Synopsis and Index
-------------------
```

Comment:
You've removed this without removing its entire content so now there is a title "Tactics" that has become a subsection of "SSReflect searching tool". Is there a reason you didn't get rid of the section entirely?

BTW, given how long this chapter is, I'm thinking the SSR maintainers will probably want to preserve this synopsis and index, but instead of repeating the syntax, maybe it could just list (with links) the SSReflect tactics, commands, flags and nonterminals.

Reply:
I haven't given much thought to what this section should eventually look like.  I removed all the `:cmd:`, `:tacn:` and `:prodn:` because they duplicate information presented earlier in the file, making it harder to do the basic syntax updating.